### ISSUE TITLE
[request] Mono 5.x instead of 4.0.4

### ISSUE THREAD

**Issue by sjorge** [TritonDataCenter/pkgsrc#46]
Can we get something from the mono 5.x series to compile on illumos?

**Comment by bushchannelstepper** [comment]
Yes please...running a VM to work around this for now isn't the best

**Comment by Toasterson** [comment]
I don't know if the 5 series received it's solaris support back but the 6.1 series has it in again. Our WIP PR here. https://github.com/OpenIndiana/oi-userland/pull/8941
